### ISSUE TITLE
Renaming tabs failing for PDF tabs

### ISSUE THREAD

**Issue by Dulskuggery** [sylouuu/chrome-tab-modifier#210]
### Tab Modifier version

**Browser:** Chrome

**Extension version:** 0.22.0

### Expected behavior

Tab to be renamed after creation of rule which gives instruction to rename the tab with a matching URL fragment

### Actual behavior

The tab renames for a fraction of a second after reloading, before reverting to the original tab name

### Steps to reproduce the behavior

1. Open a non-local PDF as a tab (e.g. by clicking a PDF link on a website to open in a new tab)
2. Copy a unique segment of the URL and head to the rule-creator tab (or rename within the tab)
3. Create desired name for tab
4. Save & reload PDF tab

**Comment by sylouuu** [comment]
Please refer to https://github.com/sylouuu/chrome-tab-modifier#known-issues

**Comment by Dulskuggery** [comment]
Thank you for your response. I selected "Allow access to file URLs" and created a new rule. Unfortunately the issue persists. Note the file is not local. I have attached an image with an example of a non-local PDF open in chrome, for which a rule exists to change the name to "Final Report". 

Regards
![Example](https://user-images.githubusercontent.com/34278448/94915090-a2cd7200-04ef-11eb-87c3-ee3ece97c2aa.png)
